language-support/hs/bindings/examples/nim-console/src/Logging.hs

```diff
@@ -0,0 +1,27 @@
+-- Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+
+
+module Logging (Mes,noMes,tagMes,colourMes,plainMes,colourWrap) where
```

Comment:
Why not `Log` instead of `Mes` throughout? e.g. `noLog`, `tagLog` etc.

Reply:
I do like `Log` better than `Mes`...

Leading  to identifiers named `log`  instead of `mes`. But I thought (wrongly) this implied the need for `import Prelude hiding(log)` frowned upon in a previous code review. Turns out this is only required if we have a top level binding for `log`.

So. In summary, I will switch from Mes -> Log !

Reply:
Internal identifiers named `l` vs `log` is not too relevant or problematic.